library/src/main/java/com/devbrackets/android/exomedia/core/listener/OnStateChangeListener.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2016 Brian Wernick,
+ * Copyright (C) 2014 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.devbrackets.android.exomedia.core.listener;
+
+import com.devbrackets.android.exomedia.core.PlaybackState;
+
+public interface OnStateChangeListener {
+    void onStateChanged(@PlaybackState int playbackState);
```

Comment:
I think it might be better to use the same value unless you see that `EmVideoView` will have more state or less state than `ExoPlayer`. If then the value can be moved internal or it can be added to this `IntDef` annotation. Having multiple variables that does the same thing is hard to manage.